lib/streamlit/elements/number_input.py

```diff
@@ -15,7 +15,7 @@
 import numbers
```

Comment:
This whole file requires a serious refactoring.

Reply:
Some `if ... else` logic could be shortened just done it 👍 Should we refactor whole file though?